web/concrete/src/Asset/AssetList.php

```diff
@@ -1,61 +1,66 @@
+    public $assets = array();
+    public $assetGroups = array();
 
-	public function getRegisteredAssets() {
-		return $this->assets;
-	}
+    public function getRegisteredAssets()
+    {
+        return $this->assets;
+    }
 
-	public static function getInstance() {
-		if (null === self::$loc) {
-			self::$loc = new self;
-		}
-		return self::$loc;
-	}
+    public static function getInstance()
+    {
+        if (null === self::$loc) {
+            self::$loc = new self;
+        }
+        return self::$loc;
+    }
 
-	public function register($assetType, $assetHandle, $filename, $args = array(), $pkg = false) {
-		$defaults = array(
-			'position' => false,
-			'local' => true,
-			'version' => false,
-			'combine' => -1,
-			'minify' => -1 // use the asset default
-		);
-		// overwrite all the defaults with the arguments
-		$args = array_merge($defaults, $args);
+    public function register($assetType, $assetHandle, $filename, $args = array(), $pkg = false)
+    {
+        $defaults = array(
+            'position' => false,
+            'local' => true,
+            'version' => false,
+            'combine' => -1,
+            'minify' => -1 // use the asset default
+        );
+        // overwrite all the defaults with the arguments
+        $args = array_merge($defaults, $args);
 
-		$class = '\\Concrete\\Core\\Asset\\' . Object::camelcase($assetType) . 'Asset';
-		$o = new $class($assetHandle);
+        $class = '\\Concrete\\Core\\Asset\\' . Object::camelcase($assetType) . 'Asset';
+        $o = new $class($assetHandle);
         if ($pkg != false) {
             if (!($pkg instanceof Package)) {
                 $pkg = Package::getByHandle($pkg);
             }
-    		$o->setPackageObject($pkg);
+            $o->setPackageObject($pkg);
+        }
+        $o->setAssetIsLocal($args['local']);
+        $o->mapAssetLocation($filename);
+        if ($args['minify'] === true || $args['minify'] === false) {
+            $o->setAssetSupportsMinification($args['minify']);
+        }
+        if ($args['combine'] === true || $args['combine'] === false) {
+            $o->setAssetSupportsCombination($args['combine']);
         }
-		$o->setAssetIsLocal($args['local']);
-		$o->mapAssetLocation($filename);
-		if ($args['minify'] === true || $args['minify'] === false) {
-			$o->setAssetSupportsMinification($args['minify']);
-		}
-		if ($args['combine'] === true || $args['combine'] === false) {
-			$o->setAssetSupportsCombination($args['combine']);
-		}
-		if ($args['version']) {
-			$o->setAssetVersion($args['version']);
-		}
-		if ($args['position']) {
-			$o->setAssetPosition($args['position']);
-		}
-		$this->registerAsset($o);
-		return $o;
-	}
+        if ($args['version']) {
+            $o->setAssetVersion($args['version']);
+        }
+        if ($args['position']) {
+            $o->setAssetPosition($args['position']);
+        }
+        if ($args['attributes']) $o->setTagAttributes($args['attributes']);
```

Comment:
Hum, I read again psr2 and I am not getting your point there.

PSR-2 does not say anything about the if one-liners … so it does not forbid it … is that the problem or am I missing something?